ui/src/components/Header/index.tsx

```diff
@@ -47,6 +49,20 @@ const Header: React.FC<Props> = (props): React.ReactElement => {
                         </div>
                     </div>
                 </div>
+                {/* Confirm email banner */}
+                {user && !user?.email && window.location.pathname !== Config.urls.verify.path && (
```

Comment:
I'd be tempted to have this in its own comp added added in the layouts rather directly in the head, but thats a preference

Reply:
I think if there were potential for it to be used anywhere else I would, but it feels really unlikely?

Reply:
😄  comes down the preference thing again i guess. I see components as just pieces of UI, not so much concerned of their reuse. Like if i was putting the header component in isolation in storybook or something, is that other bit of UI tired to the header, I dont know 🤷🏼

Reply:
As is, it's designed to only be shown in the header given the conditions. Given its minimal size and importance, I think it's probably tied to the header in this case.

I'd be inclined to keep this one as is - though I can imagine the complexity increasing in future if we want to keep using that space for other things!